python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -60,12 +60,14 @@ def __init__(self, namePattern='', localDir='', **kwds):
             namePattern (str): is the pattern of the output file that has to be written into mass storage
             localDir (str): This is the optional local directory of a file to be uploaded to mass storage
         """
+        self._checkConfig()
```

Comment:
So I think I fixed this by removing references to EOS in the config for the unit tests - I realise that the placing of the files would have been failing before but the tests would still have passed as the job would have completed successfully.

I set the path for the MassStorageFile to `/tmp` . Is that a bad idea?